tensorboard/main.py

```diff
@@ -287,7 +287,7 @@ def ConstructDebuggerPluginWithGrpcPort(context):
             ImportError(
                 err.message +
                 "\n\nTo use the debugger plugin, you need to have futures and "
-                "grpcio installed:\n  pip install futures grpcio"),
+                "grpcio installed:\n  pip install grpcio"),
```

Comment:
This removes "futures" from the requirements, but the text still says you need futures and grpcio?

Reply:
I decided to keep `futures` in the setup.py file since I'm planning on using that for things other than the debugger.

Reply:
Ok, but then shouldn't this just say "you need to have grpcio installed"?  Otherwise the mismatch between the text and the pip command shown is a bit confusing.